MaskChannels.py

```diff
@@ -190,12 +237,18 @@
     legend = TLegend(0.72, 0.72, 0.89, 0.89)
     VFAT2s_H = gDirectory.Get('%s_ID_%s_coverfHist'%(pos[iPos], port[iPos]))
     VFAT2s_S = gDirectory.Get('%s_ID_%s_coverfbychan'%(pos[iPos], port[iPos]))
+<<<<<<< HEAD
+    
+    if bool(VFAT2s_H) is False or bool(VFAT2s_S) is False:
+	continue
+=======
     VFAT2s_H.SetStats(kFALSE)
     VFAT2s_H.Draw('H SAME')
     Line_p0 = TLine(0,0,0,VFAT2s_H.GetMaximum())
     Line_p0.SetLineColor(4)
     Line_p0.Draw("SAME")
     legend.AddEntry(Line_p0, "cut if sigma = -1","L")
+>>>>>>> f69078e501fa5376408cd347169d20eaddad4cc2
```

Comment:
This is a Legacy routine.  From my understanding confChamber.py with --chConfig argument supersedes this script.

Suggest to remove MaskChannels.py completely (script was originally created at my request iirc).